pkg/volume/glusterfs/glusterfs.go

```diff
@@ -1009,5 +1016,18 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 		return nil, fmt.Errorf("StorageClass for provisioner %q must have gidMax value >= gidMin", glusterfsPluginName)
 	}
 
+	if len(parseVolumeOptions) != 0 {
```

Comment:
We are not doing parsing any more. The parsing logic was introduced when we want to do some kind of wrapping in the plugin. By wrapping I mean,  user only know about the string 'encryption', but internally we will have 'cleint.ssl on` , `server.ssl on` options tied to this string "encryption". I deviated from this path and we have agreed to move with direct passthrough of options. Thanks.